src/resources/lang/de/custom_offices.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * This file is part of SeAT
+ *
+ * Copyright (C) 2015, 2016, 2017, 2018, 2019  Leon Jacobs
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+return [
+    'planet'            => 'Planet',
+    'reinforcement'     => 'Verstärkungsphase',
```

Comment:
@ppfeufer translate that one with `Rf. Periode`
which one is better ? :)

Reply:
As mentioned in my comment behind this line, there is no real german translation for this one that actually makes sense and describes what it actually is. Verstärkungsphase is the literal translation, but does not translate what actually is meant here.

In this case I would suggest keep it as "RF. Periode" oder "RF. Zeit", since 'reinforcement' is common EVE slanf that even germans understand.